burotel/indico_burotel/client/styles/palette.scss

```diff
@@ -22,16 +22,19 @@ $highlight-color: #cf816d;
 $dark-highlight-color: #b1604b;
 $primary-color: #2185d0;
 
-$sui-red: #db2828;
-$sui-orange: #f2711c;
-$sui-green: #21ba45;
-$sui-grey: #767676;
-
-$light-red: #ffbdbd;
-
-$pre-booking-stripe-colors: ($sui-orange, lighten($sui-orange, 14%));
-$pre-booking-conflict-stripe-colors: ($sui-red, lighten($sui-red, 14%));
-$blocking-stripe-colors: (lighten($light-red, 10%), $light-red);
+// Timeline colors
+$available-color: #07e04b;
```

Comment:
can we import those from the core since they are the same?

cc @pferreir since you wrote our css import logic

Reply:
`@import 'rb_new:palette';` should work...

Reply:
> An @import loop has been found

Reply:
Ah right, I forgot we are monkeypatching this: https://github.com/indico/indico-plugins-cern/pull/36/files#diff-77289176ae9cdd3b9dac3715dab1660eR5

Reply:
So, the answer is no. 😞